regression/cbmc/xml-interface1/test.desc

```diff
@@ -4,8 +4,8 @@ CORE
 ^EXIT=10$
 ^SIGNAL=0$
 Not unwinding loop foo\.0 iteration 3 file main\.c line 5 function foo thread 0
-<result property="foo\.assertion\.3" status="SUCCESS"/>
-<result property="foo\.assertion\.1" status="FAILURE">
+<result description="assertion 0" property="foo\.assertion\.3" status="SUCCESS"/>
+<result description="assertion 0" property="foo\.assertion\.1" status="FAILURE">
```

Comment:
Nit pick: I'd prefer for this commit to be merged with prior commit that causes this test to break. Just makes any future bisecting easier.